packages/st2mistral/conf/logrotate.conf

```diff
@@ -0,0 +1,25 @@
+## Mistral API
+/var/log/mistral/mistral-api.log {
+  size 1M
+  rotate 5
+  compress
+  notifempty
+  missingok
+  create 644 mistral mistral
+  postrotate
+    pkill -USR1 -f mistral.api.wsgi
+  endscript
+}
+
+## Mistral Server
+/var/log/mistral/mistral-server.log {
+  size 10M
+  rotate 10
+  compress
+  notifempty
+  missingok
+  # Mistral doesn't handle USR1 signal right to re-open log files,
+  # Sending HUP will re-open the logs, but will stop the running workflows
+  copytruncate
+  delaycompress
+}
```

Comment:
Will run this logorate config for a several days on live `st2cicd` server to understand if it behaves OK.